communication/moderation.md

```diff
@@ -220,6 +220,19 @@ umbrella.
 - [Zoom Guidelines](./zoom-guidelines.md)
 - [Moderators](./moderators.md#zoom)
 
+### Process for Adding a Moderator
+
+This is the workflow for adding a moderator. 
+
+- Moderator applies by filing an issue in the kubernetes/community repo
+- Moderator get's 2 approvals from sponsors
```

Comment:
```suggestion
- Moderator gets 2 approvals from sponsors
```
nit -- might also be better worded as `Moderator gets approval from 2 current moderators`